docs/guidance/identifiers.md

```diff
@@ -0,0 +1,38 @@
+# Project identifiers
+
+A project identifier is a globally unique identifier for an infrastructure project. Every project in OC4IDS has a project identifier in the `id` field.
+
+Project identifiers can be used to join up information published at different times or from different systems, for example, including a project identifier in contracting data enables information on the design, construction and supervision contracts for a single infrastructure project to be joined up.
+
+## Project identifiers in contracting data
+
+With an authoritative list of infrastructure projects in a country to draw on, and if the identifiers it assigned are used within contracting systems, it would be possible to automatically check which projects have published contracting information, and to analyse that data to identify projects where further scrutiny, monitoring and data collection should take place.
+
+There are different approaches to including project identifiers in contracting data, with the best solution depending on the context of an implementation:
+
+* **Include an open field for project identifiers in procurement systems** and work with each department entering data to make sure this is populated according to some defined pattern.
+
+  This would allow some checks to be run, for example, checking that all the contracting processes over a certain value from a given agency have a project identifier, and that the identifier matches a known pattern or a local identifier list.
```

Comment:
In this and the next paragraph, remove the modal "would" (i.e. simply "allows") in the first sentence.

Use colon before "for example". Remove "known" as this is implied.

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/97/commits/6dbddcc20150fb09e92a8ab533ffde539d1f8ae7